src/img/i-fill-app--reverse.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Are these reverse icons being used anywhere? I couldn't find them in style guide.